modules/weko-admin/weko_admin/views.py

```diff
@@ -463,3 +465,35 @@ def manual_send_site_license_mail(start_month, end_month):
                                    mail_list, agg_date, data)
 
         return 'finished'
+
+
+@blueprint_api.route('/update_site_info', methods=['POST'])
+@login_required
+# @author_permission.require(http_exception=403)
+def update_site_info():
+    site_info = request.get_json()
+    format_data = format_site_info_data(site_info)
+    validate = validation_site_info(format_data)
+    if validate.get('error'):
+        return jsonify(validate)
+    else:
+        result = SiteInfo.update(format_data)
+        return jsonify(format_data)
+
+
+
+@blueprint_api.route('/get_site_info', methods=['GET'])
+@login_required
+# @author_permission.require(http_exception=403)
+def get_site_info():
+    site_info = SiteInfo.get()
+    result = dict()
+    if not site_info:
```

Comment:
Don't use not .. is.
Update code to: `if site_info is not None:`
* Need to check the others

Reply:
Hi a. Huy,
We used to use 'not ..' than 'is not None' because it can't cover the case variable is 0 or  ''.